src/Chartjs/assets/package.json

```diff
@@ -21,12 +21,12 @@
     },
     "peerDependencies": {
         "@hotwired/stimulus": "^3.0.0",
-        "chart.js": "^3.4.1 <3.9"
+        "chart.js": "^3.4.1 <3.9 || ^4.0"
```

Comment:
There's another one above under `importmap` (this is rare, weird case where the node deps vs importmap are different).

Reply:
Shoot, fixed!